src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx

```diff
@@ -25,7 +25,7 @@ class RadioField extends React.Component {
           <RadioGroup
             choices={props.choices}
             disabled={disabled}
-            value={value == '' ? null : value}
```

Comment:
Can we somehow check that value is always a string?

Reply:
i think it is since it's declared as string in proptypes and we'd get warnings otherwise